pages/installing-packages.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: page
+title: Installing Python packages with conda-forge
+permalink: /installing-packages
+nav_order: 35
+---
+
+# Avoiding the spaghetti installation
+
+Many ways to install Python and Python packages have been developed over the years, and not all of them are compatible.
```

Comment:
```suggestion
[Many ways to install Python and Python packages]( https://xkcd.com/1987/) have been developed over the years, 
and not all of them are compatible.
```

Reply:
oh, you already link in farther down, that is get for commented before reading everything.

Reply:
I like having the comic displayed, rather than linked-to, as many readers of this page would have seen it in talks. It's referring to "that problem."

So I'd rather leave this one as is.